### DISCUSSION TITLE
2.0.0-20111 Release

### DISCUSSION THREAD

**Discussion by IacobIonut01** [IacobIonut01/Gallery#267] [Announcements]
## New Features
* Added option to create new albums
* Fixed rotate button
* Added option to disable vibration feedback
* Fixed video screen timeout
* Added blacklist feature
* Fixed today listing on timeline
* Added move feature
* Fixed moving files not updating the media
* Added copy feature

## What's Changed
* Update dependency org.jetbrains.kotlin.android to v1.9.20 by @renovate in https://github.com/IacobIonut01/Gallery/pull/222
* New Crowdin updates by @IacobIonut01 in https://github.com/IacobIonut01/Gallery/pull/223
* Update dependency androidx.datastore:datastore-preferences to v1.1.0-alpha06 by @renovate in https://github.com/IacobIonut01/Gallery/pull/228
* Update dependency androidx.hilt:hilt-navigation-compose to v1.1.0 by @renovate in https://github.com/IacobIonut01/Gallery/pull/229
* Update navigation.runtime.ktx to v2.7.5 by @renovate in https://github.com/IacobIonut01/Gallery/pull/230
* Update dependency androidx.hilt:hilt-compiler to v1.1.0 by @renovate in https://github.com/IacobIonut01/Gallery/pull/231
* Update dependency composeOptions to v1.5.4 by @renovate in https://github.com/IacobIonut01/Gallery/pull/239
* Update agp to v8.2.0-rc03 - autoclosed by @renovate in https://github.com/IacobIonut01/Gallery/pull/240
* New Crowdin updates by @IacobIonut01 in https://github.com/IacobIonut01/Gallery/pull/244
* Update dependency androidx.activity:activity-compose to v1.8.1 by @renovate in https://github.com/IacobIonut01/Gallery/pull/245
* Update dependency androidx.benchmark:benchmark-macro-junit4 to v1.2.1 by @renovate in https://github.com/IacobIonut01/Gallery/pull/246
* Update material3 to v1.2.0-alpha11 by @renovate in https://github.com/IacobIonut01/Gallery/pull/247
* Update plugin baselineProfilePlugin to v1.2.1 by @renovate in https://github.com/IacobIonut01/Gallery/pull/248
* Update dependency androidx.media3:media3-exoplayer to v1.2.0 by @renovate in https://github.com/IacobIonut01/Gallery/pull/249
* Update dependency androidx.media3:media3-ui to v1.2.0 by @renovate in https://github.com/IacobIonut01/Gallery/pull/250
* New Crowdin updates by @IacobIonut01 in https://github.com/IacobIonut01/Gallery/pull/251

**Full Changelog**: https://github.com/IacobIonut01/Gallery/compare/2.0.0-20079...2.0.1-20111

<hr /><em>This discussion was created from the release <a href='https://github.com/IacobIonut01/Gallery/releases/tag/2.0.1-20111'>2.0.0-20111 Release</a>.</em>

**Comment by oreon-core** [comment]
for some reason it works at 20 fps (moto g14)
